libexec/cli/instance.list.exec

```diff
@@ -33,6 +33,7 @@ if ! USERID=`id -ru`; then
     exit 255
 fi
 
+JSONFMT="false"
```

Comment:
Also a good idea to prepend variables like this with `SINGULARITY_`.  After all, the user might want set this env var and if you export it, you want to make sure that it ends up in the registry for use in the c portion of the code.